src/cmd-remote-build

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3 -u
+
+import sys
+import argparse
+from os import environ
+from subprocess import check_call, check_output, CalledProcessError
+
+def run_cmd(command, output=None):
+    '''
+    Run the given command using check_call and verify its return code.
+    @param str command command to be executed
+    '''
+    if output:
+        try:
+            output = check_output(command.split())
+        except CalledProcessError as e:
+            print('An exception has occurred: {0}'.format(e))
+            sys.exit(1)
+        return output
+    else:
+        try:
+            check_call(command.split())
+        except CalledProcessError as e:
+            print('An exception has occurred: {0}'.format(e))
+            sys.exit(1)
+
+def build(arch, commit, file, repo):
+    '''
+    Build the image  using podman remote and push to the registry
+    @param arch
+    @param commit
+    @param file
+    @param repo
+    @param tag
+    '''
+    tag = ("%s-%s" % (arch, commit))
+    run_cmd("podman --remote build --tag %s:%s-%s -f %s" % (repo, arch, commit, file))
+    run_cmd("podman --remote push %s:%s-%s" % (repo, arch, commit))
+
+    # Podman remote doesn't allow push using digestfile. That's why the tag check is done
```

Comment:
> Podman remote doesn't allow push using digestfile. That's why the tag check is done

Is there an open issue/feature request for this? or is it documented somewhere?

Reply:
@dustymabe see https://github.com/coreos/coreos-assembler/pull/2887#discussion_r891577414